api/data_refinery_api/views.py

```diff
@@ -603,17 +593,11 @@ class ProcessorJobList(PaginatedAPIView):
 
     def get(self, request, format=None):
         filter_dict = request.query_params.dict()
-        filter_dict.pop('limit', None)
-        filter_dict.pop('offset', None)
-        jobs = ProcessorJob.objects.filter(**filter_dict)
-
-        page = self.paginate_queryset(jobs)
-        if page is not None:
-            serializer = ProcessorJobSerializer(page, many=True)
-            return self.get_paginated_response(serializer.data)
-        else:
-            serializer = ProcessorJobSerializer(jobs, many=True)
-            return Response(serializer.data)
+        limit = max(int(filter_dict.pop('limit', 100)), 100)
```

Comment:
I think it's a good idea to add a maximum for `limit` but this might affect the dashboard.

Right now we're requesting ALL Survey/Processor/Downloader jobs in a specified time interval, and then calculating their state in the front-end (pending/failed/completed/open). At some point we'll probably have many jobs and this may not be feasible.